client/commonFramework/end.js

```diff
@@ -112,6 +112,7 @@ $(document).ready(function() {
         common.displayTestResults(tests);
         if (solved) {
           common.showCompletion();
+          common.codeStorage.removeStoredValue(common.challengeName);
```

Comment:
You're right. Actually, the existing code used common.challengeName, and I just imitated that. The code in this file seems to inconsistently prepend "common." to variables that are already destructured in the const statement above. While we're looking at it I may as well go through and remove the "common." from variables that don't need it.